tests/testthat/test-expressions.R

```diff
@@ -144,6 +144,13 @@ with_mock_crunch({
         print(ds$birthyr * 3 + 5)
         print(3 * (ds$birthyr + 5))
     })
+
+    test_that("rollup functions generate expected requests", {
+        expect_identical(rollupResolution(ds$starttime), "s")
+        expect_PATCH(rollupResolution(ds$starttime) <- "M",
+            'https://app.crunch.io/api/datasets/1/variables/starttime/',
+            '{"view":{"rollup_resolution":"M"}}')
+    })
```

Comment:
Would be nice to have tests of the validation failing.

Reply:
Is this still necessary after switching to `validateResolution`? I'm assuming that function is tested

Reply:
It's tested as it's called from other functions (I should have added separate tests of `validateResolution` when I introduced it, but didn't, my bad). We should probably separate out tests for `validateResolution` and test that separately. I also am of the opinion that we should also test that `validateResolution` is being called in the more user-facing function as well — just in case something happens to the call to validate, we can catch that and correct it. The tests of `rollupResolution` don't need to cover all possible bad inputs, but at least one to ensure that we do hit `validateResolution` is good.